missions/JNK_Missions.txt

```diff
@@ -1608,12 +2323,6 @@ jnk_missions_outworld_slot = {
 			add_permanent_claim_province = {
 				province = 361
 			}
-			add_permanent_claim_province = {
-				province = 1231
-			}
-			add_permanent_claim_province = {
-				province = 1233
-			}
```

Comment:
Why remove outerworld claims?

Reply:
weird i don't remember changing that, i'll put that back